src/main/java/shapes/Shape.java

```diff
@@ -0,0 +1,43 @@
+package shapes;
+
+public class Shape {
+    private String colour;
+    private boolean filled;
+
+    //constructors
+    Shape() {
+        this.colour = "green";
+        this.filled = true;
+    }
+
+    Shape(String colour, boolean filled) {
+        this.colour = colour;
+        this.filled = filled;
+    }
+
+    String getColour() {
+        return colour;
+    }
+
+    boolean isFilled() {
+        return filled;
+    }
+
+    void setColour(String colour) {
+        this.colour = colour;
+    }
+
+    void setFilled(boolean filled) {
+        this.filled = filled;
+    }
+
+
+    @Override
+    public String toString() {
+        String isNot = "filled";
```

Comment:
Not really good name for the variable. `isNot `can be "filled" or can be "not filled" - it may confuse.

Reply:
Another way to do it:
`String filledText = isFilled() ? "filled" : "not filled";`

Reply:
> Another way to do it:
> `String filledText = isFilled() ? "filled" : "not filled";`

fixed.